Parser/asdl_c.py

```diff
@@ -947,8 +972,12 @@ def visitProduct(self, prod, name):
             fields = name+"_fields"
         else:
             fields = "NULL"
-        self.emit('state->%s_type = make_type("%s", state->AST_type, %s, %d);' %
-                        (name, name, fields, len(prod.fields)), 1)
+        self.emit(
+            'state->%s_type = make_type("%s", state->AST_type, %s, %d, %s);'
+            % (name, name, fields, len(prod.fields), reflow_c_string(asdl_of(name, prod), 1)),
+            1,
+            reflow=False,
+        )
```

Comment:
If the docstring is single-line, it may be too long. And if it is multi-line, the first line is not aligned with the rest. I suggest to break a line before docstring.
```suggestion
        docstring = asdl_of(name, prod)
        if '\n' in docstring or '(' in docstring:
            self.emit(
                'state->%s_type = make_type("%s", state->AST_type, %s, %d,'
                % (name, name, fields, len(prod.fields)),
                1,
                reflow=False,
            )
            self.emit(
                '%s);' % reflow_c_string(docstring, 2),
                2,
                reflow=False,
            )
        else:
            self.emit(
                'state->%s_type = make_type("%s", state->AST_type, %s, %d, %s);'
                % (name, name, fields, len(prod.fields), reflow_c_string(docstring, 2)),
                1,
                reflow=False,
            )
```